source/features/useful-forks.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {LinkExternalIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const downloadUrl = new URL('https://useful-forks.github.io/?repo=' + pageDetect.utils.getRepoURL());
+
+	select('#network')!.prepend(
+		<a
+			className="btn mt-2 mb-2"
+			href={downloadUrl.href}
+			target="_blank"
+			rel="noreferrer"
+		>
```

Comment:
No need for `target`
```suggestion
		<a className="btn mt-2 mb-2" href={downloadUrl.href}>
```

Reply:
Also I'd align it on the right of the title, in both cases, with a `.float-right` class. Example:

<img width="775" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/1402241/107274620-4090fa80-6a16-11eb-805d-39d6dde3c032.png">

Reply:
My personal preference would be to have it opened on a new tab. Is that generally discouraged by ``refined-github`` ?

Reply:
The `float-right` looks nice on the `Network` page, but on the `Forks` it looks like it's just randomly floating in a corner:

![image](https://user-images.githubusercontent.com/38117856/107280905-f2382780-6a26-11eb-978f-2aa3a4c2a5c0.png)

![image](https://user-images.githubusercontent.com/38117856/107280827-d59bef80-6a26-11eb-9be5-60a3bc40d0ca.png)

Reply:
> have it opened on a new tab. Is that generally discouraged by `refined-github` ?

It's discouraged everywhere. Generally links don't need to open in a new tab unless they break the current flow (e.g. the user is about to buy something or fields haven't been saved). The user can choose where to open the links

Reply:
> but on the `Forks` it looks like it's just randomly floating in a corner

Yeah that page is rather empty, but generally we try to avoid content shift (i.e. adding a button and pushing links lower, which can cause misclicks).